<launch>

    <arg name="octomap_file" default="$(find esdf_map_generator)/octo_binary/octomap.bt" />
    <!-- 设置终点 -->

    <param name="goal_x" value="13.0" />
    <param name="goal_y" value="-7.0" />
    <param name="goal_z" value="1.5" />

    <param name="is_2D_planning" value="false" />

    

    <node name="path_planner" pkg="esdf_map_generator" type="path_planning_demo" output="screen">
            <param name="octomap_file" value="$(arg octomap_file)" />

    </node>


    <node type="rviz" name="rviz" pkg="rviz" args="-d $(find esdf_map_generator)/rviz/esdf_gen.rviz" ></node>


  </launch>
